modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -462,7 +462,19 @@
          "REVIEW_STATUS": "Review status",
          "PREVIOUS": "Previous",
          "NOCONTENT": "No data available",
-         "PUBLISHED": "Published"
+         "PUBLISHED": "Published",
+         "HOVER": {
```

Comment:
Maybe "TOOLTIP" would be a better choice

Reply:
done